scipy/stats/tests/test_distributions.py

```diff
@@ -2172,6 +2172,11 @@ def test_lomax_accuracy():
     # regression test for gh-4033
     p = stats.lomax.ppf(stats.lomax.cdf(1e-100,1),1)
     assert_equal(p, 1e-100)
-
+    
+def test_rayleigh_accuracy():
+    # regression test for gh-4034
+    p = stats.rayleigh.isf(stats.rayleigh.sf(9,1),1)
+    assert_equal(p, 9.0)
```

Comment:
Exact equality is still not guaranteed; for comparing floating point numbers you should normally use a tolerance. I suggest to replace this by ``assert_almost_equal(p, 9.0, decimal=15)``.